lib/routes/ehentai/search.js

```diff
@@ -1,16 +1,19 @@
+const querystring = require('querystring');
 const EhAPI = require('./ehapi');
 
 module.exports = async (ctx) => {
     const page = ctx.params.page;
     let params = ctx.params.params;
-    const bittorrent = ctx.params.bittorrent || false;
+    const routeParams = querystring.parse(ctx.params.routeParams);
```

Comment:
The `querystring` API is considered Legacy. While it is still maintained, new code should use the `URLSearchParams` API instead.
docs: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams